docs/deployment/pulumi/aws.md

```diff
@@ -0,0 +1,188 @@
+---
+id: aws
+title: Deploying Backstage on AWS Lightsail
+sidebar_label: AWS
+description: How to deploy Backstage on AWS Lightsail
+---
+
+> **DISCLAIMER: The `deploy` command is in alpha and still experimental. Do not use the `deploy` command for production deployments.**
+
+Getting started with Backstage mostly starts with a proof-of-concept phase. To make the proof-of-concept phase more seamless there is the `deploy` command. The `deploy` command should make it easier for you to deploy your Backstage instance on a cloud provider. Currently the `deploy` command only supports deploying `Backstage` on AWS Lightsail.
+
+## What is AWS Lightsail
+
+:::tip
+
+AWS offers a free tier for up to three months on $10 USD/month Container service (Micro -1 node). By default we use the `nano` node, so if you are a new user this approach shouldn't cost you anything. For more information, refer to the [pricing](https://aws.amazon.com/lightsail/pricing/) documentation.
+
+:::
+
+AWS Lightsail offers a simple way to run containers in the cloud. To learn more about AWS Lightsail, please refer to the [official documentation](https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-container-services-deployments).
+
+## Creating user in AWS
+
+- Open the AWS console and navigate to the IAM section
+- In the left side menu click on `Users` and then click on `Add users`
+- Specify an username and then click on `Next`
```

Comment:
```suggestion
- Specify a username and then click on `Next`
```

Generally `an` is used if the first sound of the next word sounds like a vowel - however, the first sound of `username` sounds like `yew`, and `y` is a consonant in the `a`/`an` rule.
English is fun :/